tests/h/util/markdown_test.py

```diff
@@ -18,11 +18,11 @@ def test_it_ignores_inline_match(self):
 
     def test_it_sanitizes_the_output(self, markdown_render, sanitize):
         markdown.render("foobar")
-        sanitize.assert_called_once_with(markdown_render.return_value)
+        sanitize.assert_called_once_with(markdown_render.return_value.return_value)
 
     @pytest.fixture
     def markdown_render(self, patch):
-        return patch("h.util.markdown.markdown")
+        return patch("h.util.markdown._get_markdown")
```

Comment:
?

Reply:
`markdown` was the global variable used for caching, `_get_markdown` is the (now lru_cached) function.

Reply:
Right gotcha, it was a bit hard to piece together in the previous PR. I'd totally forgotten about that by this point.